portfolio/src/main/webapp/script.js

```diff
@@ -27,64 +28,55 @@ function generateImageUrl() {
  * functionality but with a new, random image. 
  */
 function newImageButton() {
-
     const pennyButton = document.getElementById('penny-button');
-    const currentImgUrl = pennyButton.getAttribute("src");
+    const currentImgUrl = pennyButton.getAttribute('src');
     let newImgUrl = currentImgUrl;
 
-    //ensure new image is not the same as old one 
+    /* ensure new image is not the same as old one */ 
     while (newImgUrl === currentImgUrl) {
         newImgUrl = generateImageUrl();
     }
 
     const pennyContainer = document.getElementById('penny-container');
-    let newHTML = `<input id="penny-button" type="image" src="${newImgUrl}"\
-     onClick="newImageButton()"/>`;
+    let newHTML = `<input id="penny-button" type="image" src="${newImgUrl}"` +
+        ` onClick="newImageButton()"/>`;
     pennyContainer.innerHTML = newHTML;
 }
 
 /**
-  * Automatically fill gallery page with all available images
+  * Automatically fills gallery page with all available images
   */
```

Comment:
As we're getting nitty, a couple things to consider: 

1. 'Automatically' is redundant and can be removed. 
2. This ideally would have a period at the end of the sentence.
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
adding that right now!